src/System.IO/tests/Stream/Stream.netstandard17.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class StreamMethods
+    {
+        [Fact]
+        public void CreateWaitHandle()
+        {
+            using (var str = CreateStream())
+            {
+                ManualResetEvent first = str.CreateWaitHandle();
+                ManualResetEvent second = str.CreateWaitHandle();
```

Comment:
I honestly didn't know this method existed.  Just looked up the implementation. What a useless method :) Ugh.